documentation/ci-fix.md

```diff
@@ -123,6 +123,11 @@ If that guidance is not enough, please also refer to the [LintDiff rules.md doc]
 
 To reproduce LintDiff failures locally, see [CONTRIBUTING.md / How to locally reproduce a LintDiff failure occurring on a PR](https://github.com/Azure/azure-openapi-validator/blob/main/CONTRIBUTING.md#how-to-locally-reproduce-a-lintdiff-failure-occurring-on-a-pr).
 
+## Swagger LintDiff for TypeSpec
+
+### `Record<unkown>` causes `AvoidAdditionalProperties` and `PropertyTypeObjectNoDefinition`
+The use of `Record<unkown>` in TypeSpec is discouraged, and there is a TypeSpec lint rule to enforce this.  If you still need to use `Record<unknown>`, the swagger generated will cause many LintDiff errors of types `AvoidAdditionalProperties` and `PropertiesTypeObjectNoDefinition`.  You will need to suppress both the TypeSpec violation (in TypeSpec source code) and the LintDiff violations (in `readme.md`).
```

Comment:
You can link folks to https://aka.ms/pr-suppressions for guidance on how to suppress things.

Also, consider writing "OpenAPI" instead of "Swagger" inside the text body.

Reply:
Shouldn't the guidance for suppressions be in the general "LintDiff: section, not specific to TypeSpec?

Why use "OpenAPI" instead of "Swagger"?  The name of the check is "Swagger LintDiff".

Reply:
To my knowledge Swagger is an obsolete / legacy / deprecated name. We didn't get around to changing the name of the check due to technical considerations.

Reply:
Re:

> Shouldn't the guidance for suppressions be in the general LintDiff section, not specific to TypeSpec?

I mean, you were mentioning people should suppress things, so I was thinking to plop down the link right there, as a convenience for the reader.

Reply:
> Re:
> 
> > Shouldn't the guidance for suppressions be in the general LintDiff section, not specific to TypeSpec?
> 
> I mean, you were mentioning people should suppress things, so I was thinking to plop down the link right there, as a convenience for the reader. That suppressions aka.ms link is for all the validation checks we run, not only LintDiff.

I think the "how-to-fix" guidance in the section above is sufficient.